tests/utils_tests/test_lorem_ipsum.py

```diff
@@ -21,3 +80,22 @@ def test_paragraphs(self):
                 'deserunt mollit anim id est laborum.'
             ]
         )
+
+    def test_paragraphs_not_common(self):
+        """
+        Not common lorem ipsum paragraph
+        """
+        para = paragraphs(1, False)
+        self.assertTrue(len(para) > 0)
+        self.assertNotEqual(
+            para, [
+                'Lorem ipsum dolor sit amet, consectetur adipisicing elit, '
```

Comment:
perhaps this should be moved to a class constant so it doesn't have to be repeated.